From 924fe4c46d4ca80d619019c698c23989f6cb5c0a Mon Sep 17 00:00:00 2001
From: markw65 <mark_st@myosotissp.com>
Date: Sun, 22 Jan 2023 09:45:27 -0800
Subject: [PATCH] Remove bogus BitmapResource

---
 source/YetAnotherWatchFaceView.mc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/source/YetAnotherWatchFaceView.mc b/source/YetAnotherWatchFaceView.mc
index a0bcaa7..ecac168 100644
--- a/source/YetAnotherWatchFaceView.mc
+++ b/source/YetAnotherWatchFaceView.mc
@@ -50,7 +50,7 @@ class YetAnotherWatchFaceView extends Ui.WatchFace
 		var deviceSettings = Sys.getDeviceSettings();
         if(deviceSettings has :requiresBurnInProtection) {    
             _isCanBurn = deviceSettings.requiresBurnInProtection;
-			_checkmateImage = Application.loadResource( Rez.Drawables.checkmate ) as BitmapResource;
+			_checkmateImage = Application.loadResource( Rez.Drawables.checkmate );
         }
 		
     }
-- 
2.34.1

