From 31e17c35995342147945be797d7a4b86e9966a65 Mon Sep 17 00:00:00 2001
From: markw65 <mark_st@myosotissp.com>
Date: Sun, 22 Jan 2023 10:21:16 -0800
Subject: [PATCH] Fix imports

---
 source/AuxHRApp.mc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/AuxHRApp.mc b/source/AuxHRApp.mc
index 3049edf..5d734e6 100644
--- a/source/AuxHRApp.mc
+++ b/source/AuxHRApp.mc
@@ -1,4 +1,5 @@
 using Toybox.Application as App;
+import Toybox.Ant;
 
 class AuxHRApp extends App.AppBase {
 	var mSensor;
-- 
2.34.1

