From 32c2d5ce537c457fdc5b4083b27367101810c122 Mon Sep 17 00:00:00 2001
From: markw65 <mark_st@myosotissp.com>
Date: Sun, 22 Jan 2023 10:23:20 -0800
Subject: [PATCH] Fix imports

---
 source/model/Match.mc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/model/Match.mc b/source/model/Match.mc
index 3a0e2ad..6c0e215 100644
--- a/source/model/Match.mc
+++ b/source/model/Match.mc
@@ -3,6 +3,7 @@ using Toybox.ActivityRecording;
 using Toybox.Activity;
 using Toybox.FitContributor;
 using Toybox.WatchUi;
+import Toybox.Lang;
 
 enum Player {
 	YOU = 1,
-- 
2.34.1

